app/controllers/coronavirus_landing_page_controller.rb

```diff
@@ -4,16 +4,15 @@ class CoronavirusLandingPageController < ApplicationController
   slimmer_template "gem_layout_full_width"
 
   def show
+    @content_item = content_item.to_hash
```

Comment:
Is `content_item` already a hash by this point?

Reply:
I just moved the code up from line 15, it doesn't appear to be a hash from my testing of the code